packages/powersync_flutter_libs/README.md

```diff
@@ -0,0 +1,3 @@
+# powersync_flutter_libs
+
+### Flutter binaries for [Powersync](https://pub.dev/packages/powersync) please go there for documentation.
```

Comment:
It would be good to include an overview/link to where we get the binaries from, e.g. a link to powersync-sqlite-core github.